flusher/flusher/feeds_db.py

```diff
@@ -0,0 +1,98 @@
+import sqlalchemy as sa
+import enum
+
+from .db import (
+    metadata,
+    Column,
+    CustomDateTime,
+)
+
+PRICE_HISTORY_PERIOD = 60 * 60 * 24 * 7 * 1e9  # 1 week
+
+# Define the SignalPriceStatus Enum
+class SignalPriceStatus(enum.Enum):
+    Unspecified = 0
+    Unsupported = 1
+    Unavailable = 2
+    Available = 3
+
+class CustomSignalPriceStatus(sa.types.TypeDecorator):
+    impl = sa.Enum(SignalPriceStatus)
+
+    def process_bind_param(self, value, dialect):
+        return SignalPriceStatus(value)
+
+# Define the PriceStatus Enum
+class PriceStatus(enum.Enum):
+    Unspecified = 0
+    UnknownSignalID = 1
+    NotReady = 2
+    Available = 3
+    NotInCurrentFeeds = 4
+
+class CustomPriceStatus(sa.types.TypeDecorator):
+    impl = sa.Enum(PriceStatus)
+
+    def process_bind_param(self, value, dialect):
+        return PriceStatus(value)
+
+feeds_signal_prices_txs = sa.Table(
+    "feeds_signal_prices_txs",
+    metadata,
+    Column("transaction_id", sa.Integer, sa.ForeignKey("transactions.id"), primary_key=True),
+    Column("validator_id", sa.Integer, sa.ForeignKey("validators.id"), primary_key=True),
+    Column("feeder_id", sa.Integer, sa.ForeignKey("accounts.id")),
+    Column("timestamp", CustomDateTime, index=True),
+    sa.Index("ix_signal_prices_txs_validator_id_transaction_id", "validator_id", "transaction_id"),
+    sa.Index("ix_validator_id_timestamp", "validator_id", "timestamp"),
+)
```

Comment:
don't think it should have table name in the index like below.

Reply:
add table name